test.html

```diff
@@ -2,26 +2,74 @@
 <html lang="en-US">
   <head>
     <meta charset="utf-8" />
-    <title>hello-wasm example</title>
+    <title>read-fonts-wasm example</title>
     <style>
-        svg { width: 45%; }
+      main {
+        max-width: 1000px;
+        margin-left: auto;
+        margin-right: auto;
+      }
+      .svg-list {
+        display: flex;
+        flex-wrap: wrap;
+        align-items: center;
+        gap: 10px;
+      }
+      svg {
+        max-width: 100px;
+      }
     </style>
   </head>
   <body>
     <script type="module">
-      // Don't have woff2_decompress in Rust at time of writing and as far as I can tell the browser
-      // doesn't expose it to js
```

Comment:
There is also a module for woff2 reads in allsorts.
https://github.com/yeslogic/allsorts/blob/332e29d8667f928bceb123833c139e885226399f/src/woff2.rs#L1

Reply:
I want to try that one but I'll merge this first.